src/data/color-fonts.js

```diff
@@ -0,0 +1,79 @@
+define(function () {
+  return {
+    name: "Color fonts",
+    description: "Four color font formats are currently in existence: three OpenType proposals, one proprietary. Support is fragmented. When unsupported, the browser will fall back to regular glyphs in the font.",
+    specification: "http://pixelambacht.nl/2014/multicolor-fonts/",
```

Comment:
I think it is a great idea to link to your article, but the specification field is reserved for specifications. The description field supports markdown, so that would be a great place to link to it.